### ISSUE TITLE
Megaman 3 (USA) hash is wrong

### ISSUE THREAD

**Issue by pcone** [SmokeMonsterPacks/EverDrive-Packs-Lists-Database#117]
The CRC32 in the SMDB file is 452D8089, but it looks like the No Intro rom is BE6DBB5D

**Comment by pcone** [comment]
I think my copy of nointro is just wrong

**Comment by frederic-mahe** [comment]
@pcone you are right though. The SMDB hash values are different from [No-Intro](
https://datomatic.no-intro.org/?page=show_record&s=45&n=1401)'s.

It could be due to a bit difference in the NES header, or to the absence of a header.

**Comment by pcone** [comment]
Interestingly Romhacking.net seems to have 452D8089 as the no-intro hash (https://www.romhacking.net/translations/3264/)

**Comment by pcone** [comment]
Mega Man 2 is different from here too: https://datomatic.no-intro.org/?page=show_record&s=45&n=1398 - the SMDB file has 5e268761

**Comment by maniac79** [comment]
Hi, I tried to understand the issue, but I failed.
However, I believe I can prove that this issue can be closed without action.

**Facts**
1. I checked Mega Man 3 (USA).nes from 2 different/recent No-Intro ROM sets from the web (20200127-000518 & 20200329-092100) and both match with the hash in the SMDB (452D8089).
2. In the latest DAT (downloaded today) the hash reads as "603AAA57".
3. I used clrmamepro to verify the 2 ROMs against the DAT and they matched. <- I don't understand this part... are the hashes in the No-Intro DAT obscured in some strange way? Why?
4. Odd finding: There's an entry on VIMM that supposedly has the same hash as listed in the DAT file (603AAA57). Upon inspection, it has the same hash from the SMDB as well (452D8089).

Considering all facts, it seems the hash in the SMDB for Mega Man 3 is correct.
Same for Mega Man 1 + 2 btw - that were the only ones I checked, but I believe my statements above are valid for every game.

Thus, I believe this issue can be closed without action.
Although I would like to understand the reason, maybe someone can explain.
